cluster-autoscaler/core/scale_up.go

```diff
@@ -238,10 +239,14 @@ func (sr *skippedReasons) Reasons() []string {
 
 var (
 	backoffReason         = &skippedReasons{[]string{"in backoff after failed scale-up"}}
-	maxLimitReachedReason = &skippedReasons{[]string{"max limit reached"}}
+	maxLimitReachedReason = &skippedReasons{[]string{"max size reached"}}
```

Comment:
maybe "max node group size reached"? I don't think it's obvious.

Reply:
Done